src/logic/contracts/safeContractErrors.ts

```diff
@@ -5,12 +5,14 @@ import { getWeb3 } from 'src/logic/wallets/getWeb3'
 import { GnosisSafe } from 'src/types/contracts/gnosis_safe.d'
 import { logError, Errors } from '../exceptions/CodedException'
 
-export const decodeMessage = (message: string): string => {
+const GENERIC_FAILED_TX_MSG = 'unable to decode contract error message'
```

Comment:
Maybe "transaction failed for an unknown reason"?

Reply:
I find "unknown" very mysterious. This message sheds more light into it. Should I change it anyway?

Reply:
Let's keep it if you think it's better.